airflow/providers/amazon/aws/operators/base_aws.py

```diff
@@ -85,10 +86,12 @@ def __init__(
         region_name: str | None = None,
         verify: bool | str | None = None,
         botocore_config: dict | None = None,
+        region: str | None | ArgNotSet = NOTSET,  # Required for `.partial` signature check
```

Comment:
We add it for backward compact.
Shouldn't we raise deprecation warning on it in favor of `region_name`?

Reply:
It will raise deprecation warning, we have a test for that. 
But what I've found that we could simplify constructors

Reply:
But better simplify into the separate PR